### ISSUE TITLE
Fix `property-no-unknown` false positives for `corner-shape`

### ISSUE THREAD

**Issue by regseb** [stylelint/stylelint#9065]
### What is the problem you're trying to solve?

Stylelint reports that the new [`corner-shape`](https://developer.mozilla.org/Web/CSS/Reference/Properties/corner-shape) property is unknown. And also `corner-top-left-shape`, `corner-top-right-shape`, `corner-bottom-left-shape`, `corner-bottom-right-shape`...

### What solution would you like to see?

Add these new properties to the [`property-no-unknown`](https://stylelint.io/user-guide/rules/property-no-unknown/) rules.

**Comment by ybiquitous** [comment]
@regseb Thanks for the report. This issue appears to be a false-negative bug in the rule. Can you reproduce the bug by using the Stylelint demo?

**Comment by jeddy3** [comment]
Thanks for the report. Reproducible in [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYLICcB2NMFoBnACyQAcZ4UAjGANxgBsBuFAXxABoQAzAS0YwAckgC2cRDAAeYsoIB0YQoS7gI2fgHMEIaWQiYALilg8kAV0bHgAHWwoU0wzGxRCVANoByQoYCegox82Ib4kBp8mkSGSK5ImFBeALqcdmzMdqrhWgBiBqJIhjq+AUzBhorqWvKiAFYq3LBkKoi29ig2ICWB5Z1UnYyFML6dqe2d3WUhYVWR0bFQ8VB9HSCDziMgaZlsQA) with:

```css
a { corner-shape: bevel; }
```

The [`known-css-properties` upstream package](https://www.npmjs.com/package/known-css-properties) was last published 8 months ago.

`corner-shape` is in the [latest csstree patch](https://github.com/csstools/postcss-plugins/blob/98fdb9550fa49768bcf49d413479a7e4e791503c/packages/css-syntax-patches-for-csstree/dist/index.json#L228):

```
"corner-shape": "<'corner-top-left-shape'>{1,4}",
```

Shall we use this opportunity to replace `known-css-properties` with the csstree lexer matcher for this rule?

**Comment by jeddy3** [comment]
Given the thumbs up, I've labelled the issue as ready to implement. 

@regseb Please consider [contributing](https://stylelint.io/contributing) if you have time. There are [steps on how to fix a bug in a rule](https://stylelint.io/contributor-guide/rules#fix-a-bug-in-a-rule) in the Contributor guide.

The [`at-rule-descriptor-no-unknown`](https://github.com/stylelint/stylelint/blob/main/lib/rules/at-rule-descriptor-no-unknown/index.mjs) rule is an example of a similar one that uses the csstree lexer matcher, and can be used as a reference.

**Comment by Mouvedia** [comment]
> Shall we use this opportunity to replace known-css-properties with the csstree lexer matcher for this rule?

👎 

I didn't check but I would expect a lot of regressions.
